### DISCUSSION TITLE
How change Footer Github Link

### DISCUSSION THREAD

**Discussion by ekaone** [shuding/nextra#60] [General]
Hi, how I can change Github link footer on the bottom right corner that refer to my repository..?, is there another option property at  from `theme.config.js` ..?, thanks in advance

![Screenshot 2020-12-16 192510nextra](https://user-images.githubusercontent.com/50127577/102348495-9330d600-3fd4-11eb-864b-38db35e25446.png)

**Comment by ekaone** [comment]
Done...with these configs at `theme.config.js`, thanks for amazing stuff 💖

```
repository: 'https://github.com/shuding/nextra',
docsRepository: 'https://github.com/shuding/nextra',
branch: 'master',
path: '/',
titleSuffix: ' – Nextra',
nextLinks: true,
prevLinks: true,
search: true,
darkMode: true,
footer: true,
footerText: 'MIT 2020 © Nextra.',
footerEditOnGitHubLink: true,
```

**Reply by ProxyJoshua** [reply]
Is there a way to hide the github icon on the top-right?

**Reply by skaiwlkr** [reply]
@ProxyJoshua Yeah there is a way. Just leave the string on repository (theme.config.js) empty.

`repository: ''`
